.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "scripts"]
 	path = scripts
 	url = https://github.com/OpenVicProject/scripts
+[submodule "deps/lexy-vdf"]
+	path = deps/lexy-vdf
+	url = git@github.com:OpenVicProject/lexy-vdf.git
```

Comment:
This needs to be HTTPS, like the other URLs in the file, otherwise anyone using git without SSH setup won’t be able to update/initialise the submodule.